src/aat/java/uk/gov/hmcts/cmc/claimstore/tests/functional/citizen/SealedClaimPdfTest.java

```diff
@@ -0,0 +1,57 @@
+package uk.gov.hmcts.cmc.claimstore.tests.functional.citizen;
+
+import org.junit.Before;
+import org.junit.Test;
+import uk.gov.hmcts.cmc.claimstore.tests.functional.BaseSealedClaimPdfTest;
+import uk.gov.hmcts.cmc.claimstore.utils.Formatting;
+import uk.gov.hmcts.cmc.domain.models.Address;
+import uk.gov.hmcts.cmc.domain.models.Claim;
+import uk.gov.hmcts.cmc.domain.models.amount.AmountBreakDown;
+import uk.gov.hmcts.cmc.domain.models.sampledata.SampleClaimData;
+
+import java.io.IOException;
+import java.util.function.Supplier;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class SealedClaimPdfTest extends BaseSealedClaimPdfTest {
+
+    @Before
+    public void before() {
+        user = idamTestService.createCitizen();
+    }
+
+    @Test
+    public void shouldBeAbleToFindTestClaimDataInClaimIssueReceiptPdf() throws IOException {
+        shouldBeAbleToFindTestClaimDataInPdf("claimIssueReceipt");
```

Comment:
claimIssueReceipt is not a sealed claim, not sure that you are benefitting from inheritance here.

Reply:
This was in the original `SealedClaimPdfTest`. Remove the `Sealed` prefix?